### PR TITLE
Arrange and add missing keys for cy locales

### PR DESCRIPTION
**Pull request by hmzasif**
This PR fixes the error messages by moving them under the `errors:` section to fix the test cases. It also adds missing keys.

### CODE REVIEW

File: rails/locale/cy.yml [hunk 1]
```diff
@@ -171,22 +166,29 @@ cy:
...
       too_long: yn rhy hir (cewch %{count} llythyren ar y fwyaf)
       too_short: yn rhy fyr (rhaid am o leiaf %{count} llythyren)
       wrong_length: gyda maint anghywir o lythrennau (dylai fod yn %{count} llythyren)
     template:
       body: 'Cafwyd broblemau gyda''r meysydd canlynol:'
       header:
+        zero: Atalwyd y %{model} hwn rhag ei gadw gan 0 nam
         one: Atalwyd y %{model} hwn rhag ei gadw gan 1 nam
         two: Atalwyd y %{model} hwn rhag ei gadw gan %{count} nam
+        few: Atalwyd y %{model} hwn rhag ei gadw gan ychydig nam
```
Comment (sunny) [inline rails/locale/cy.yml:191]: Shouldn’t this still have the `count` variable?

```suggestion
        few: Atalwyd y %{model} hwn rhag ei gadw gan %{count} nam
```

**Review by sunny — APPROVED** [review]
Thanks! Could you add an entry to the CHANGELOG as well?

**Comment by hmzasif** [comment]
> Thanks! Could you add an entry to the CHANGELOG as well?

You are welcome, I have added it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,7 @@
   - Portuguese (pt-BR): am/pm
   - Romanian (ro): am/pm
   - Swedish (sv sv-FI sv-SE): am/pm
+  - Welsh (cy): Move the keys to rught place and add missing keys
```
Comment (pama) [inline CHANGELOG.md:20]: There is small typo here "rught" should "right"

Comment (hmzasif) [reply CHANGELOG.md]: Sorry for it; it's already been fixed.

**Pull request merged**